### ISSUE TITLE
Pushed resources from production websites cause false h2spec failures

### ISSUE THREAD

**Issue by joakime** [summerwind/h2spec#91]
Regarding ... https://twitter.com/tunetheweb/status/988522917532184580

> To be honest I didn’t install it and just ran it against the webtide website (Jetty 9.4.9 according to server header): 81/146 (55%)

The overwhelming majority of those failures are because hitting https://webtide.com/ will result in pushed resources that h2spec doesn't like seeing.

**Comment by joakime** [comment]
Perhaps allow for path to be hit on server as well?
So you can avoid requesting resources that will result in PUSH_PROMISE frames?

eg:

```
$  ./h2spec --insecure --strict --host=webtide.com --port=443 --path=/favicon.ico --tls --verbose
```

**Comment by summerwind** [comment]
Thank you for the report.
It seems good to add the option to specify path. I'll add the option.

**Comment by summerwind** [comment]
Fixed in [v2.2.0](https://github.com/summerwind/h2spec/releases/tag/v2.2.0).
